common/djangoapps/student/helpers.py

```diff
@@ -496,7 +509,7 @@ def _cert_info(user, course_overview, cert_status):
         return default_info
 
     status = template_state.get(cert_status['status'], default_status)
-    is_hidden_status = status in ('unavailable', 'processing', 'generating', 'notpassing', 'auditing')
```

Comment:
Is it there defensively maybe? Like just in case it stops being an impossible status? Or is that a dumb question (I'm not embedded in this cert code like you are).

Reply:
I think it could easily be added back in when it stops being impossible.

The big part here is that it is doing `template_state.get` and `unavailable` isn't in the `template_state` so it can't happen. And the `default_status` is `processing`. So there is just no way `status` could equal it at that point. I don't know what happened!